web-ui/src/components/checkin/__snapshots__/CheckinHistory.test.js.snap

```diff
@@ -0,0 +1,3 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`renders correctly 1`] = `<div />`;
```

Comment:
This doesn't look like a useful snapshot. I would think there has to be a way to mock that call and wait for the re-render, no? I'd do some digging on that. An empty div snapshot is kinda pointless.